packages/nx/src/utils/package-manager.ts

```diff
@@ -142,6 +144,20 @@ export function getPackageManagerCommand(
         list: 'npm ls',
       };
     },
+    bun: () => {
+      return {
+        install: 'bun install',
+        ciInstall: 'bun install --no-cache',
+        updateLockFile: 'bun install --frozen-lockfile',
+        add: 'bun install',
+        addDev: 'bun install -D',
+        rm: 'bun rm',
+        exec: 'bun',
```

Comment:
I wasn't sure whether to use the --bun flag to force inclusion in bun. However, although bun isn't exactly a 1-to-1 match yet, I thought it could introduce too many non-issues to NX.

Maybe 1 to follow up with eventually

Reply:
@Jordan-Hall Where would the `--bun` flag go? It should be consistent with other package managers.

Reply:
exec: 'bun --bun'
dlx: 'bunx --bun',
        run: (script: string, args: string) => `bun run --bun ${script} -- ${args}`,


It basically stops node shebang  and forces it to bun